bats/fb-katello-client.bats

```diff
@@ -7,8 +7,16 @@ load os_helper
 load foreman_helper
 load fixtures/content
 
-@test "enable katello-agent" {
-  foreman-installer --foreman-proxy-content-enable-katello-agent true
+setup() {
+  KATELLO_VERSION=$(tKatelloVersion)
+}
+
+@test "enable remote execution (or katello-agent for Katello 4.9 and below)" {
+  if tIsVersionNewer "${KATELLO_VERSION}" 4.10; then
```

Comment:
you could put something like:

```bash
tIsKatelloAgentRemoted() {
  KATELLO_VERSION=$(tKatelloVersion)
  tIsVersionNewer "${KATELLO_VERSION}" 4.10
}
```

into the shared files and use that instead of `if newer than 4.10` all the time, but up to you :)